src/YaccConstructor/Program.fs

```diff
@@ -44,6 +44,32 @@ let log (e:System.Exception) msg =
     "\nMessage:\n  " + msg
     |> printfn "%s"
 
+open Argu
+
+type CLIArguments =
+    | [<Inherit; AltCommandLine("-f","-frontend")>] F of feName:string option
```

Comment:
Seems that Argu uses constructor name as default key name. So you can move full names from AltCommandLine attribute to constructor name.